AzurePSDriveStorageAccount.psm1

```diff
@@ -1,6 +1,6 @@
 ﻿using namespace Microsoft.PowerShell.SHiPS
 
-$script:Azure_Storage = if($IsCoreCLR){'AzureRM.Storage.NetCore'}else{'Azure.Storage'}
+$script:Azure_Storage = if($IsCoreCLR){'Azure.Storage.NetCore'}else{'Azure.Storage'}
```

Comment:
Did they change the name of this NetCore module?

Reply:
No a typo I made it at the first checkin ages ago.